src/OnlineSales/Controllers/ActivityLogController.cs

```diff
@@ -0,0 +1,60 @@
+﻿// <copyright file="ActivityLogController.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.Web;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Nest;
+using OnlineSales.Configuration;
+using OnlineSales.Data;
+using OnlineSales.DTOs;
+using OnlineSales.Entities;
+using OnlineSales.Infrastructure;
+
+namespace OnlineSales.Controllers;
+
+[Authorize]
+[Route("api/[controller]")]
+public class ActivityLogController : ControllerBase
+{
+    private readonly IMapper mapper;
+    private readonly IOptions<ApiSettingsConfig> apiSettingsConfig;
+    private readonly ElasticClient elasticClient;
+    private readonly IConfiguration configuration;
+
+    public ActivityLogController(IConfiguration configuration, IMapper mapper, IOptions<ApiSettingsConfig> apiSettingsConfig, EsDbContext esDbContext)
+    {
+        this.mapper = mapper;
+        this.apiSettingsConfig = apiSettingsConfig;
+        this.configuration = configuration;
+        elasticClient = esDbContext.ElasticClient;
+    }
+
+    [HttpGet]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
+    public virtual async Task<ActionResult<List<ActivityLogDto>>> Get([FromQuery] string? query)
```

Comment:
To be consequent with existing naming policy, please rename ActivityLogDto to ActivityLogDetailsDto.